src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    static ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Don't use static methods or static fields in your solution. Instead, create an instance of the class. Static methods and fields can lead to issues with object-oriented design, especially when dealing with state that should be encapsulated within an instance.